Topic3TDD/src/exc4/RomanConverterTest.java

```diff
@@ -0,0 +1,29 @@
+package exc4;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class RomanConverterTest {
+
+	@Test
+	public void test1() {

```

Comment:
As I said before, it's ok, but a good practice in tests is to try to declare in the name of the method what are you testing for example in this case testConvertVFromRomanToInt